test/itkPairwiseTestHelper.hxx

```diff
@@ -36,7 +36,7 @@ double
 calculateError(const itk::TileConfiguration<Dimension> &    stageTiles,
                const itk::TileConfiguration<Dimension> &    actualTiles,
                const std::string &                          inputPath,
-               int                                          paddingMethod,
+               uint8_t                                          paddingMethod,
```

Comment:
alignment

Reply:
I ran clang-format, so all style is conformant :nail_care: